package.json

```diff
@@ -119,6 +119,7 @@
     "swr": "^2.0.3",
     "tailwindcss-animate": "^1.0.7",
     "tailwindcss-radix": "^2.7.0",
+    "valibot": "^0.32.0",
```

Comment:
I've added [Valibot](https://app.opensauced.pizza/s/fabian-hiller/valibot) to the project. It is MIT licensed and a small package (1kb). It's used for some validation in this PR and I plan to use it elsewhere in the application long term.

This is similar to Zod, but with a smaller footprint, and the `pipe` function is 🔥

I know we have Zod in the API, but this is completely separate, and depending how deep Zod usage is in the API, we may want to consider Valibot over there as well.